src/components/Book.js

```diff
@@ -2,22 +2,31 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Book = (props) => {
-  const { book } = props;
+  const { book, remove } = props;
+
+  const handleRemove = () => {
+    remove(book);
+  };
   return (
     <tr>
       <td>{book.id}</td>
       <td>{book.title}</td>
       <td>{book.category}</td>
+      <td>
+        <button type="button" onClick={handleRemove}>Remove Book</button>
+      </td>
     </tr>
   );
 };
 
 Book.defaultProps = {
   book: {},
+  remove: () => {},
 };
 
 Book.propTypes = {
   book: PropTypes.objectOf(PropTypes.object),
+  remove: PropTypes.func,
 };
```

Comment:
- [ ] [OPTIONAL] The remove function not being passed as a prop would basically break one of the functionalities of the app, right?
That means it should be a required prop, with no need of a default prop value.